hathor/conf/settings.py

```diff
@@ -458,6 +458,15 @@ def from_yaml(cls, *, filepath: str) -> 'HathorSettings':
             validators=_VALIDATORS
         )
 
+    def get_default_capabilities(self) -> list[str]:
+        """Return the default capabilities."""
+        return [
```

Comment:
Should we make it a tuple so it's immutable?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575

Reply:
Thanks!